Makefile

```diff
@@ -45,11 +49,59 @@ $(wasm_out): $(wasm_src) $(sdk_srcs)
 	@echo "Building WASM module..."
 	@tinygo build -o $@ -target=wasi $<
 
-test_scripts := $(shell realpath $(wildcard ./test/scripts/*.txtar))
+# We manage the rendered template directory explicitly as the rendered templates
+# may contain secrets. Use `/tmp` if `TMPDIR` is not set.
+rendered_template_dir := $(or $(TMPDIR), /tmp)/bky-basm-rendered
+
+.PHONY: make-rendered-template-dir
+make-rendered-template-dir:
+	@echo "rendered templates: $(rendered_template_dir)"
+	@mkdir -p $(rendered_template_dir)
+
+.PHONY: delete-rendered-template-dir
+delete-rendered-template-dir:
+	@rm -rf $(rendered_template_dir)
+
+# Specify all the files that are templates to be rendered
+template_src := $(wildcard $(template_dir)/*.mustache)
+# Specify all the files that we expect to exist after rendering as renames of the template files
+template_out := $(patsubst $(template_dir)/%.mustache,$(rendered_template_dir)/%,$(template_src))
+# Mark the rendered files as intermediate. Make will delete them after the build
+# if they still exist.
+.INTERMEDIATE: $(template_out)
```

Comment:
I made the management of the rendered templates temp directory a little more explicit so we can clean up the dir after certain tasks.

Reply:
moot point now, removed the rendering of templates to disk.